CMakeLists.txt

```diff
@@ -161,32 +163,34 @@ ELSE (GIT_BINARY)
 	ENDIF (NOT GIT_FOUND)
 ENDIF (GIT_BINARY)
 
-IF (PREFIX)
+# PREFIX has no sence in case of win32
+IF (PREFIX AND NOT WIN32)
 	ADD_DEFINITIONS(-DPREFIX=${PREFIX})
-ENDIF (PREFIX)
+ENDIF (PREFIX AND NOT WIN32)
 
 # Set build types and display info
-	MESSAGE(STATUS "DYNAMIC_LINKING               : ${DYNAMIC_LINKING}\n")
 IF(RELEASE)
 	MESSAGE(STATUS "CMAKE_INSTALL_PREFIX          : ${CMAKE_INSTALL_PREFIX}")
 	MESSAGE(STATUS "BIN_PREFIX                    : ${BIN_PREFIX}")
 	MESSAGE(STATUS "DATA_PREFIX                   : ${DATA_PREFIX}")
+    IF(LOCALIZE)
+        MESSAGE(STATUS "LOCALE_PATH                   : ${LOCALE_DIR}")
+    ENDIF(LOCALIZE)
 	MESSAGE(STATUS "DESKTOP_ENTRY_PATH            : ${DESKTOP_ENTRY_PATH}")
 	MESSAGE(STATUS "PIXMAPS_ENTRY_PATH            : ${PIXMAPS_ENTRY_PATH}")
 	MESSAGE(STATUS "PIXMAPS_UNITY_ENTRY_PATH      : ${PIXMAPS_UNITY_ENTRY_PATH}")
 	MESSAGE(STATUS "MANPAGE_ENTRY_PATH            : ${MANPAGE_ENTRY_PATH}\n")
 	SET(CMAKE_BUILD_TYPE RELEASE)
+	ADD_DEFINITIONS(-DRELEASE)
 	# FIXME: Removed -Werror for a while. Currently it is broken in upstream.
 	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Os -std=c++11 -MMD -Wall -Wextra")
 	IF(MINGW)
 		# Workaround for: http://ehc.ac/p/mingw/bugs/2250/
 		ADD_DEFINITIONS(-D__NO_INLINE__)
 	ENDIF(MINGW)
-	IF(NOT WIN32 AND NOT PREFIX)
-		# PREFIX has no sence in case of win32
-		ADD_DEFINITIONS(-DDATA_DIR_PREFIX=${CMAKE_INSTALL_PREFIX})
-	ENDIF(NOT WIN32 AND NOT PREFIX)
-	ADD_DEFINITIONS(-DRELEASE)
+#	IF(NOT WIN32 AND NOT PREFIX)
```

Comment:
Forget to remove comment. PREFIX on Windows build controlled on line 167:
```+# PREFIX has no sence in case of win32
IF (PREFIX AND NOT WIN32)
 	ADD_DEFINITIONS(-DPREFIX=${PREFIX})
ENDIF (PREFIX AND NOT WIN32)
So it NEVER will be used on Windows build. But i am not sure about meaning of WIN32 var.

Right now i am not sure about DDATA_DIR_PREFIX, so i left legacy behaviour.